apps/fulfillment-e2e/.env.template

```diff
@@ -1 +1,2 @@
-ORYX_FULFILLMENT_BACKEND_URL=
\ No newline at end of file
+ORYX_FULFILLMENT_BACKEND_URL=
```

Comment:
Can you please add description for variables.

Reply:
description is in readme file